lib/redux/middleware/sentry_api_middleware.dart

```diff
@@ -45,27 +58,40 @@ class SentryApiMiddleware extends MiddlewareClass<AppState> {
             store.dispatch(FetchOrgsAndProjectsProgressAction(
                 '${organization.name}: Checking for sessions...',
                 ++currentProgress / fullProgress));
+
+            final projectIdsSpan =
+                transaction.startChild('task', description: 'projectIds');
+
             try {
               final projectsWithSessionsForOrganization =
                   await api.projectIdsWithSessions(organization.slug);
               projectIdsWithSessions
                   .addAll(projectsWithSessionsForOrganization);
             } catch (e) {
+              projectIdsSpan.throwable = e;
               if (e is ApiError && e.statusCode == 400) {
+                projectIdsSpan.status = SpanStatus.notFound();
                 Sentry.addBreadcrumb(Breadcrumb(
                     message: 'Org has no projects -> $e',
                     level: SentryLevel.error));
               } else {
+                projectIdsSpan.status = SpanStatus.internalError();
                 rethrow;
               }
+            } finally {
+              projectIdsSpan.finish();
```

Comment:
Is there a reason you never await `transaction.finish();` calls?

Reply:
No, just didn't notice it. Would like to have an option so the compiler would give me a hint.

Reply:
You can try to enable this lint: https://dart-lang.github.io/linter/lints/unawaited_futures.html